### ISSUE TITLE
stream to youtube/periscope

### ISSUE THREAD

**Issue by arnauldb** [iizukanao/picam-streamer#22]
Hello,

Thank you for picam. Is it possible to stream to youtube or periscope?

**Comment by iizukanao** [comment]
@arnauldb Yes, it's possible but you should directly run picam instead of using picam-streamer. Please refer to this instructions for uploading to Ustream. https://github.com/iizukanao/picam#publishing-to-ustream The same applies to YouTube. I will be happy to help you.

**Comment by arnauldb** [comment]
Thank you @iizukanao , I used picam and ffmpeg and it works perfectly.
